node/standalone/src/service.rs

```diff
@@ -72,7 +93,10 @@ pub fn new_partial(
 	>,
 	ServiceError,
 > {
-	let inherent_data_providers = sp_inherents::InherentDataProviders::new();
+	// TODO this author id should not be hard coded. This is just for simple testing. We need to
+	// wire this back to the CLI.
+	let example_author = H160::from_low_u64_le(0x0123456789abcdef);
+	let inherent_data_providers = build_inherent_data_providers(example_author)?;
```

Comment:
@JoshOrndorff this method sets the timestamp inherent and it is reset to the `MockTimestampInherentDataProvider` a few lines further below https://github.com/PureStake/moonbeam/pull/173/files#diff-8e0ad4bf45adc6c17ae99038981286a2538f17e5430d0e64155f19bf2f701f7bR115 Does the second registered timestamp inherent provider overwrite the first?

Reply:
Oh wow, I forgot all about that. That explains some confusion I had last Friday. Good catch. I don't really have an expectation for what would happen if they are both registered, but I definitely didn't mean to do it that way.